README.md

```diff
@@ -135,3 +135,26 @@ sequenceDiagram
         Note over A,PM: SpendPermission data
     end
 ```
+
+## Security
+
+Audited by [OpenZeppelin](https://spearbit.com/)
+
+<table border="1">
+    <tr>
+        <th>Audit</th>
+        <th>Report</th>
```

Comment:
Add extra column for `Date` for just when the report was finalized. That would be 10/29 for #1 and 12/10 for #3

Reply:
| Audit | Date | Report |
|--------|---------|---------|
| First private audit | 10/29/2024 | [Report](https://cantina.xyz/portfolio/57d76772-97ae-4807-9930-ca6ad4b3b753) |
| Public Competition Audit | | |
| Final private audit| 12/10/2024 | [Report](https://cantina.xyz/portfolio/d948aa16-c8e5-48a2-885e-f6fa99762f2b) |


How's this?